pluma/pluma-view.c

```diff
@@ -56,6 +56,8 @@
 
 #define PLUMA_VIEW_GET_PRIVATE(object)(G_TYPE_INSTANCE_GET_PRIVATE ((object), PLUMA_TYPE_VIEW, PlumaViewPrivate))
 
+static gboolean middledown = FALSE;
```

Comment:
I think it would be better to add this variable to the private struct (`PlumaViewPrivate`) and use a bit more descriptive name, like `middle_button_pressed`.